com/bl/ttt/tictactoe.py

```diff
@@ -97,7 +106,7 @@ def process_valid_move(self, move):
         if self._is_computer_game and \
            self.is_computer_turn() and not \
            self.is_game_over():
-            self.get_computer_move()
+            self._get_computer_move()
 
```

Comment:
Might be good to log the state of _player_move_log and move counter at the end of process_valid_move

Reply:
Done